Build/Customizations/Server/Extensions.Server.targets

```diff
@@ -70,7 +70,6 @@
   <Target Name="Server_CIBuild_Chrome" DependsOnTargets="TestBuild_ChromeConfiguration;CheckProperties;SetVersionCiAndNightlyBuild;TestBuild" />
```

Comment:
`Remotion.Web.Development.WebTesting.CoypuElementScopeExtensions.IsTransparent` still has an IE workaround, TBD if it really only affects IE.

Reply:
Good point. they would be equivalent. I'll update the docs but leave it in.